README.md

```diff
@@ -50,7 +50,18 @@ import (
 And then in your main function initialize the metrics
 
 ``` go
-amImpl.Init(nil, am.DefBuckets)
+	// Everything in BuildInfo is optional.
+	// You can also use any string variable whose value is
+	// injected at build time by ldflags.
```

Comment:
Could you document how that would work? That seems like it would be the recommended way of doing this, no?